app/Http/Controllers/CommentController.php

```diff
@@ -7,28 +7,40 @@
 use App\Models\Post;
 use App\Models\Comment;
 use App\Models\Activity;
+use App\Models\User;
+use App\Notifications\CommentNotification;
 
 class CommentController extends Controller
 {
     public function comment(Request $request)
     {
-        $comment = new Comment;
         $post = Post::with('user')->findOrFail($request->postId);
-        $comment->content = $request->valueComment;
-        $comment->user_id = Auth::user()->id;
-        $comment->post_id = $request->postId;
-        
-        $comment->save();
-        $activities = new Activity;
-        $activities->user_id = Auth::user()->id;
-        $activities->action = config('status_acotion.comment');
-        $activities->notify = json_encode([
+        $comment = Comment::create([
+            'content' => $request->valueComment,
+            'user_id' => Auth::user()->id,
+            'post_id' => $request->postId,
+        ]);
+
+        $data_activities = [
             'message' => 'profile.action_comment_history',
             'user_name' => $post->user->username,
             'post_id' => $post->id,
-        ]);
+        ];
+        $activities = new Activity;
+        $activities->user_id = Auth::user()->id;
+        $activities->action = config('status_acotion.comment');
+        $activities->notify = json_encode($data_activities);
         $comment->activities()->save($activities);
 
+        $user_get_noti = User::findOrFail($post->user->id);
```

Comment:
xem lại cách đặt tên biến
```suggestion
        $notificationsOfUser = User::findOrFail($post->user->id);
```